packages/web3-types/src/apis/eth_execution_api.ts

```diff
@@ -104,10 +104,9 @@ export type TransactionUnsignedAPI =
 	| TransactionLegacyUnsignedAPI;
 
 // https://github.com/ethereum/execution-apis/blob/main/src/schemas/transaction.yaml#L211
-export type TransactionSignedAPI =
-	| Transaction1559SignedAPI
-	| Transaction2930SignedAPI
-	| TransactionLegacySignedAPI;
+export type TransactionSignedAPI = Transaction1559SignedAPI &
+	Transaction2930SignedAPI &
+	TransactionLegacySignedAPI;
```

Comment:
I think this type is better like this:
```suggestion
export type TransactionSignedAPI = (
	| Transaction1559SignedAPI
	| Transaction2930SignedAPI
	| TransactionLegacySignedAPI
) |
	Partial<Transaction1559SignedAPI & Transaction2930SignedAPI & TransactionLegacySignedAPI>;
```
 Or what do you all think?

Reply:
it was changes yesterday